features/upgrade-insecure-requests.yml

```
@@ -0,0 +1,7 @@
+name: Upgrade insecure requests
+description: The `Upgrade-Insecure-Requests` HTTP header instructs a user agent to upgrade insecure resource requests to secure transport before fetching them.
```

The spec and docs make this sound like a _request_ header. I think it's something like this?

```suggestion
description: The `Upgrade-Insecure-Requests` HTTP request header tells the server that the response should redirect to a secure (HTTPS) resource.
```